tensorflow/python/ops/variable_scope_test.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
you **need** to add test cases; but put them in kernel_tests/variable_scope_test; not in this new file.